package.json

```diff
@@ -30,8 +33,135 @@
     "autocomplete.provider": {
       "description": "Intelligent Ruby code completion",
       "versions": {
-        "1.1.0": "provideAutocompletion"
+        "2.0.0": "provideAutocompletion"
       }
     }
+  },
+  "readme": "# Atom Autocomplete+ Ruby Suggestions\n[![Dependency status](https://david-dm.org/ianhattendorf/autocomplete-ruby.svg)](https://david-dm.org/ianhattendorf/autocomplete-ruby)\n[![GitHub version](https://badge.fury.io/gh/ianhattendorf%2Fautocomplete-ruby.svg)](http://badge.fury.io/gh/ianhattendorf%2Fautocomplete-ruby)\n\nProvides intelligent code completion for Ruby. Requires [RSense](https://github.com/rsense/rsense) and [Autocomplete+](https://github.com/atom-community/autocomplete-plus).\n\n## Status\nCurrently pre-alpha, buggy and not fully implemented. All help is welcome.\n\n## Why?\nBecause I wanted Ruby code completion in Atom and [atom-rsense](https://github.com/rsense/atom-rsense) hasn't been updated since June 2014, which is a long time for an Atom package. It's currently broken and I didn't like the dependency on Opal, as much as I enjoy Ruby, so I decided to create this.\n\n## Installation\nInstall required gems and Atom packages:\n```shell\n$ gem install rsense\n$ apm install autocomplete-plus\n```\n\n## Usage\nLaunch the rsense server in the project root you want completion:\n```shell\n$ rsense start\nrsense version: 0.5.18\nlogs at: /tmp/rsense.log\nprocess running at: 925\n```\n\n## Bugs\nA lot. If you're brave enough to try this out and notice any specifically, feel free to open an issue or even better submit a pull request.\n\n## Development\nClone the repository into your working directory:\n```shell\n$ git clone git@github.com:ianhattendorf/autocomplete-ruby.git\n```\n\nInstall dependencies:\n```shell\n$ cd autocomplete-ruby\n$ apm install\n```\n\nLink to Atom as a dev package:\n```shell\n$ apm link --dev\n```\n\nFeel free to fork it and submit a pull request for any changes you make.\n",
```

Comment:
Do we need all of these extra fields? AFAIK npm/apm automatically locate README.md.